internal/handlers/pg/pgdb/databases.go

```diff
@@ -30,6 +30,20 @@ import (
 // validateDatabaseNameRe validates FerretDB database / PostgreSQL schema names.
 var validateDatabaseNameRe = regexp.MustCompile("^[a-zA-Z_-][a-zA-Z0-9_-]{0,62}$")
 
+// DatabaseExists returns true if FerretDB database / PostgreSQL schema exists and false otherwise.
+func DatabaseExists(ctx context.Context, tx pgx.Tx, db string) (bool, error) {
```

Comment:
Please use existing `Databases` function. It is not worth adding another top-level and exported helper function into already bloated package for a single usage.

In general, that package should become smaller, not bigger